public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuItem.tsx

```diff
@@ -44,3 +46,22 @@ export const PanelHeaderMenuItem: FC<Props & PanelMenuItem> = props => {
     </li>
   );
 };
+
+export function getDropdownLocationCssClass(element: HTMLElement | null) {
```

Comment:
Ah, we do similar things across multiple places. Think we finally need to create some solid abstraction around positioning elements.

Reply:
yea, I know I tried to but could not share it in the end

Reply:
No worries, I think we can have that as a separate task as we need to unify this across dropdowns, selects etc